DEPRECATIONS.md

```diff
@@ -4,6 +4,7 @@ This document lists deprecated constructs in the SDK, with their expected time 
 
 | API / Feature | Deprecated (warning) | Deprecated (error) | Removed | Notes |
 | --- | --- | --- | --- | --- |
+| `StreamDimens` constructor containing params: `attachmentsContentImageHeight`  | 2022.08.16 <br/>5.8.0 | 2022.08.30 ⌛ | 2022.09.13 ⌛ | This constructor has been deprecated. Use the constructor that does not contain these parameters. |
```

Comment:
Worded as a plural because the next PR will deprecate two more parmeters regarding Giphys when auto-sizing Giphys is implemented. 

They will be added here.

Reply:
We should probably update this given that we're shelvign giphy improvements for a bit